deploy/release.groovy

```diff
@@ -15,15 +17,13 @@ def ci (){
     }
 
     stage('Functional Tests'){
-        dir('runtime'){
-            container('ui'){
-                sh '''
-        npm cache clean --force
-        npm install
-        cd src/tests/functionalTests
-        DEBUG=true HEADLESS_MODE=true ./run_ts_functional_tests.sh smokeTest
-'''
-            }
+        container('ui'){
+            sh '''
+            npm cache clean --force
+            npm cache verify
+            npm install
```

Comment:
Isn't this npm install supposed to run inside `runtime`?
I see you've removed the `dir(runtime)`  https://github.com/fabric8-ui/fabric8-planner/pull/2419/files#diff-318c73c5bdb7e830d5e72aefe23c7df6L18 line as well.

This command will run in the planner root directory

Reply:
Yes, that is correct. That section runs ´./scripts/run-functests.sh´ from the root, not in runtime.

Reply:
Mmmh..the headless mode runs fine on my machine. I also did not change something there, this basically stayed as it was before, just at another place.